File: src/main/scala/avalon/friend/FriendMessageHandler.scala
package avalon.friend

import avalon.extend.Recorder
import avalon.util.FriendMessage

/**
	* Created by Eldath Ray on 2017/4/1 0001.
	*
	* @author Eldath Ray
	*/
object FriendMessageHandler {
	def handle(message: FriendMessage): Unit = {
		// if (MessageChecker.checkEncode(message)) return;
		Recorder.getInstance.recodeFriendMessage(message)
	}
}

File: src/main/kotlin/avalon/friend/FriendMessageHandler.kt
package avalon.friend

import avalon.function.Recorder
import avalon.tool.pool.Constants
import avalon.tool.pool.Constants.Basic.LANG
import avalon.util.FriendMessage

object FriendMessageHandler {
	fun handle(message: FriendMessage) {
		if (!Constants.Basic.DEBUG)
			Recorder.getInstance().recodeFriendMessage(message)
		message.response(LANG.getString("friend.handler.reply"))
	}
}